l3build-install.lua

```diff
@@ -294,5 +294,5 @@ function install_files(target,full,dry_run)
 end
 
 function install()
-  return install_files(gethome(),options["full"],options["dry-run"])
+  return install_files(gethome(),Opts.full,Opts["dry-run"])
```

Comment:
This is the kind of inconsistency I mentioned in an earlier commit: The syntax suggests a difference between `full` and `dry-run` where no such difference exists.